templates/base.html

```diff
@@ -76,131 +76,139 @@
-                                    <span class="glyphicon glyphicon-log-in" aria-hidden="true"></span>
-                                    {% trans "Log in"%}
-                                </a>
-                            </li>
-                            {% endif %}
-                        </ul>
-
-                        {# Separate menu optimized for mobile users #}
-                        <ul class="nav navbar-nav visible-xs">
-                            {% notification_menu %}
-                            {% group_select %}
-                            {% block mobilemenu %}{% endblock %}
-                            <li role="presentation" class="header"><h4>{% trans "Site" %}</h4></li>
-                            <li role="presentation">
-                                <a href="{% url 'home' %}">
-                                    <span class="glyphicon glyphicon-home" aria-hidden="true"></span>
-                                    {% trans "Home" %}
-                                </a>
-                            </li>
-                            {% course_menu %}
-                            {% if user.is_authenticated %}
-                            <li role="presentation">
-                                <a href="{% url 'profile' %}">
-                                    <span class="glyphicon glyphicon-user" aria-hidden="true"></span>
-                            	    {{ user.first_name }} {{ user.last_name }}
-                                </a>
-                            </li>
-                            <li role="presentation">
-                                <a href="{% url 'logout' %}">
-                                    <span class="glyphicon glyphicon-log-out" aria-hidden="true"></span>
-                            	    {% trans "Log out"%}
-                                </a>
-                            </li>
-                            {% else %}
-                            <li role="presentation">
-                                <a href="{% url 'login' %}">
-                                    <span class="glyphicon glyphicon-log-in" aria-hidden="true"></span>
-                                    {% trans "Log in" %}
-                                </a>
-                            </li>
-                            {% endif %}
-                        </ul>
+                                {% else %}
+                                <li role="presentation">
+                                    <a href="{% url 'login' %}">
+                                        <span class="glyphicon glyphicon-log-in" aria-hidden="true"></span>
+                                        {% trans "Log in" %}
+                                    </a>
+                                </li>
+                                {% endif %}
+                            </ul>
+                        </div>
                     </div>
-                </div>
-            </nav>
+                </nav>                    
```

Comment:
However, you can clean the spaces at the end of the line though (probably left from before).